x/mint/keeper/keeper.go

```diff
@@ -315,25 +321,33 @@ func (k Keeper) GetEmissionInfo(ctx context.Context) (*types.Params, *types.Even
 	if err != nil {
 		return nil, nil, errorsmod.Wrap(err, "failed to get blocks per month")
 	}
+	startingEmissionsBlockHeight, err := k.GetStartingEmissionsBlockHeight(ctx)
+	if err != nil {
+		return nil, nil, errorsmod.Wrap(err, "failed to get starting emissions block height")
+	}
 
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
 	blockHeight := uint64(sdkCtx.BlockHeight())
-	numberOfRecalcs := blockHeight / blocksPerMonth
-	blockHeightTarget_e_i_LastCalculated := numberOfRecalcs*blocksPerMonth + 1          //nolint:revive // var-naming: don't use underscores in Go names
-	blockHeightTarget_e_i_Next := blockHeightTarget_e_i_LastCalculated + blocksPerMonth //nolint:revive // var-naming: don't use underscores in Go names
+	blockHeightTGEAdjusted := blockHeight - (uint64(startingEmissionsBlockHeight))
```

Comment:
```suggestion
	blockCountSinceTGE := blockHeight - (uint64(startingEmissionsBlockHeight))
```

The naming seems a bit misleading to me, this is not a height

Reply:
yep, changed.